src/mesh/Default.h

```diff
@@ -29,6 +29,7 @@
 #else
 #define default_ringtone_nag_secs 60
 #endif
+#define default_network_ipv6_enabled true
```

Comment:
Let's set the default to false so we can merge this in as a non-breaking change. It's been too long in review - sorry about that.